watcher-c/readme.md

```diff
@@ -0,0 +1,34 @@
+# Using the watcher library from C
+
+The watcher can also be used from C by including the `watcher-c.h` file on compilation and in your project.
+
+You can cross compile the library via the `watcher-c/cross-compile.sh` script or download the precompiled library from the releases.
+
+### Compiling the library from source
+
+If you want to compile the library yourself you can follow the following example:
+
+First check out the `next` branch, compile a shared `libwatcher-c.so` library with `gcc` and install
```

Comment:
This will be the main branch at some point soon

Reply:
Good point, I changed it to mention that you can checkout any branch or release branch